feature/platform/healthz/tests/status/README.md

```diff
@@ -0,0 +1,103 @@
+# Health-1.2: Healthz component status paths
+
+## Summary
+
+Validate healthz status paths exist for select OC component types.  There
+are two operational use cases for this test.
+
+1. As a network operator, I want to know if a device is healthy.  If the
+   device is unhealthy, I may choose to execute a mitigation or repair action.
+   The choice of action is influenced by which component(s) are not healthy.
+2. As a SDN controller, I want to know if a device is ready to be programmed
+   for traffic forwarding.
+
+## Testbed type
+
+* [Single DUT](https://github.com/openconfig/featureprofiles/blob/main/topologies/dut.testbed)
+
+## Testbed prerequisites
+
+There are no DUT configuration pre-requisites for this test.  The DUT must
+contain the following component types:
+        "CONTROLLER_CARD",
+        "LINECARD",
+        "FABRIC",
+        "POWER_SUPPLY",
+        "INTEGRATED_CIRCUIT"
+
+The DUT should have a HEALTHY state for all the above components.
+
+## Procedure
+
+* Healthz-1.2.1: Validate healthz status
+  * gNMI Subscribe to the /components tree using ON_CHANGE option.
+  * Validate `/components/component/healthz/state/status` returns `HEALTHY`
+    for each of the following component types:
+        "CONTROLLER_CARD",
+        "LINECARD",
+        "FABRIC",
+        "POWER_SUPPLY",
+        "INTEGRATED_CIRCUIT"
+  * Validate the following paths return a valid value:
+    * /components/component/healthz/state/last-unhealthy
```

Comment:
There should be a step to transition the components to an unhealthy state first to ensure that the last-unhealthy time is initialized. E.g. by issuing a gnoi reboot for a component.